tool/site/server_hapi/src/services/communication/mock-eps-client.ts

```diff
@@ -50,6 +50,30 @@ export class MockEpsClient implements EpsClient {
     return Promise.resolve({statusCode, fhirResponse, spineResponse: spineResponseStr})
   }
 
+  async makeReleaseRequest(body: Parameters): Promise<EpsResponse<Bundle | OperationOutcome>> {
+    const url = `https://${process.env.APIGEE_DOMAIN_NAME}/electronic-prescriptions/FHIR/R4/Task/$release`
+    const statusCode = 200
+    //TODO - why is the mock client sending real requests?
+    const spineResponse = (await axios.post(url, body, {
+      headers: {
+        "X-Request-ID": uuid.v4(),
+        "X-Raw-Response": "true"
+      }
+    })).data
+    const spineResponseStr = typeof spineResponse === "string" ? spineResponse : JSON.stringify(spineResponse)
```

Comment:
The live client doesn't stringify the response if it's an object, so I think we can just return spineResponse here too

Reply:
updated on next PR